docs/concepts/package-types.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Types of packages
+parent: Concepts
+---
+
+| Type       | `modular start` | `modular build` | `modular test` | `modular lint` | Custom index / assets      | Bundled                                      | Entrypoint                                                                                                                                                                                                                                                                                       |
+| ---------- | --------------- | --------------- | -------------- | -------------- | -------------------------- | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `app`      | Yes             | Yes             | Yes            | Yes            | Yes                        | Yes                                          | The compiled bundle will use `src/index.tsx` as an entrypoint and include it in the output `index.html`                                                                                                                                                                                          |
+| `esm-view` | Yes             | Yes             | Yes            | Yes            | No                         | Yes + rewriting external dependencies to CDN | The compiled bundle will use `src/index.tsx` as an entrypoint and [link it](../esm-views/output-package-manifest.md) in the output `package.json`; the entrypoint needs to default export a React component for `modular start`/ [standalone `index.html`](../esm-views/how-to-build.md) to work |
```

Comment:
Good point, but I prefer "Yes. Will rewrite external dependencies to point to a CDN" because by default Modular rewrites them and bundling external dependencies has unintuitive ramifications (described in the "known limitations" section) and is an escape hatch that needs to be used only in rare situations.